packages/ui/src/components/Modal/index.tsx

```diff
@@ -211,6 +211,7 @@ const MemoDisclosure = memo(Disclosure)
 const StyledDialogBackdrop = styled(DialogBackdrop)`
   display: flex;
   position: fixed;
+  z-index: 9999;
```

Comment:
would it worth it to add kind of zindex constant file to keep some tracks of the differents index and make sure all make sense ? if bug fix is urgent we can merge like this and add a comment may be ?

Reply:
I've done as I've seen in other places. But actually you're right, I will prefer constants that can defined some "layers". So it will be easier to manage layers between each other. 

For now, I just can define a `LAYER_TOP` or something like that and set it up to 9999 (it will be then use as our top level layer). Wdyt?

Reply:
On a long term vision we are planing to remove all z-index as each portal element should be shown depending on when it appears on screen. That said, here the issue is mainly due to the way we handle the modal with reakit and we are also planing on removing reakit to do something homemade. This is a working solution for now and I don't know if it will be worth defining constant for z-index in the project.